src/test/java/Problem2Test.java

```diff
@@ -1,14 +1,22 @@
 import org.junit.Test;
-
+//importing junit test
 import static org.junit.Assert.*;
-
+//importing assert library
 public class Problem2Test {
-
-    // Example test
-    // Okay to change
     @Test
     public void testBubbleSort() {
-        assertTrue(false);
-        // add your tests here
+        int[] [] data= {{1,3,2,4,5},{1,3,2},{2,1},{1}};
```

Comment:
good test!

consider add a few more for the edge cases such as empty array, null array, etc.